tests/unit/PushNotificationsTest.js

```diff
@@ -0,0 +1,47 @@
+import Onyx from 'react-native-onyx';
+import PushNotification from '../../src/libs/Notification/PushNotification';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import waitForPromisesToResolve from '../utils/waitForPromisesToResolve';
+
+// The Onyx handler for subscribing and unsubscribing is in this module
+// so we must import it
+import '../../src/libs/actions/Session';
```

Comment:
Thanks for writing tests 🙇 This test is great - but not sure if it's _really_ a unit test for the `PushNotifications` lib. We are testing the code in the actions file.

**non-blocking:** I would suggest incorporating this test into the existing `SessionTest.js` file. We have a `TestHelper.signInWithTestUser()` method that will simulate signing in the user and calls the action which signs in the user.

Probably we could:

- Figure out some way to call `Session.signOut()`
- Sign in a user with `signInWithTestUser()` (check the mock like we are doing in this test)
- Sign out a user (check the mock again)